internal/csi-addons/networkfence/fencing_test.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2022 The Ceph-CSI Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package networkfence
+
+import (
+	"sort"
+	"testing"
+)
+
+func TestGetIPRange(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		cidr        string
+		expectedIPs []string
+	}{
+		{
+			cidr:        "10.23.56.67/32",
+			expectedIPs: []string{"10.23.56.67"},
+		},
+		{
+			cidr:        "62.76.47.12/30",
```

Comment:
why did you pick this address? Use addresses from https://datatracker.ietf.org/doc/html/rfc1918

Reply:
Updated now  :+1: